plume-models/src/safe_string.rs

```diff
@@ -101,3 +101,15 @@ impl AsRef<str> for SafeString {
         &self.value
     }
 }
+
+use rocket::request::FromFormValue;
+use rocket::http::RawStr;
+
+impl<'v> FromFormValue<'v> for SafeString {
+    type Error = &'v RawStr;
```

Comment:
can someone help me understand what `'v` means?

Reply:
`'v` is a lifetime, because you got a reference to a `RawStr`, not a `RawStr` directly. This helps Rust figure out how long a struct containing pointers (references) can live, and when some parts of it might got dropped and it's therefore no longer safe to access a struct. You can read more about it in [rust's book](https://doc.rust-lang.org/1.9.0/book/lifetimes.html), which probably explain better than me :wink:

Reply:
okay, thanks! i was just not clear on the notation